src/ScrollSync.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Why is this needed?

Reply:
Ah this was probably from my eslint config....
```suggestion
```

Reply:
So I'm confused now... without this line the ESLint is failing but you said you don't know why. Let's find out before the rule will be disabled please. I don't see any specific changes that would cause this so maybe there is a possible bug that left undetected?